style.css

```
@@ -0,0 +1,85 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+}
+
+nav {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.hamburger {
+  background-color: #b400;
+  border: none;
+}
+
+.logo {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.main {
+  background-image: url("./img/header-shapes\ mobile.png");
+  background-size: 100% 80vh;
+  background-repeat: no-repeat;
+  background-color: #fff;
+  margin-bottom: 20px;
+  border-radius: 0 0 0 100px;
+  min-height: 100vh;
+  box-sizing: border-box;
+}
+
+.primary-text {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  padding: 50px 24px 0 24px;
+  margin-top: 114px;
+}
```

- [ ] **[OPTIONAL]:** You did great with this section, however, you are required to use flexbox to align your content. I'm sorry that the reviewer didn't point this out, but I strongly suggest you make this change since flexbox gives you a lot of options to design responsive layouts! 🤓 🎨 

```suggestion
.main {
  background-image: url("./img/header-shapes\ mobile.png");
  background-repeat: no-repeat;
  background-color: #fff;
  margin-bottom: 20px;
  border-radius: 0 0 0 100px;
  box-sizing: border-box;
  height: calc(100vh - 55px); /* Here, 55px represents the height of the toolbar */
  display: flex;
  flex-direction: column;
  justify-content: center;
  gap: 12px;
}

.primary-text {
  font-family: 'Poppins', sans-serif;
  font-style: normal;
  font-weight: 700;
  font-size: 40px;
  line-height: 52px;
  color: #172b4d;
  padding: 50px 24px 0 24px;
}
```